### ISSUE TITLE
Demo not working on Unity 5.6?

### ISSUE THREAD

**Issue by orlyliba** [brianasu/unity-ray-marching#10]
Thank you for sharing your code.
Does the demo work on Unity 5.6?
When I open the scene I don't see a volume.
Also, perhaps you can explain the usage:
"Set your volume target size..."  --> is this the size of the cube?
"Drag all your slices...into the Slices" --> where is "Slices"?
Thank you

**Comment by charmaine** [comment]
@orlyliba Yes, it should work! If you click on the Main Camera and look in the Inspector, you should see a section for slices- make the size 99 and drag the slices in there (in order). The volume target size can be set by changing the numbers of "Volume texture size" right under the slices.
